src/Database/LSMTree/Internal/Snapshot.hs

```diff
@@ -39,6 +42,11 @@ import           Database.LSMTree.Internal.UniqCounter (UniqCounter,
 import           System.FS.API (HasFS)
 import           System.FS.BlockIO.API (HasBlockIO)
 
+-- | Custom text to include in a snapshot file
+newtype SnapshotLabel = SnapshotLabel Text
```

Comment:
A newtype is good but do we need `Text` here? This is the only thing that pulls in the extra `text` package dep.

Reply:
Never mind, adding the dep on `cborg` in #443 adds `text` anyway.